grunttasks/generate-valid-styles.js

```diff
@@ -0,0 +1,42 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+var path = require('path');
+var stripSpecialChars = require('./strip-special-chars');
+var VALID_STYLES_FILE_NAME = 'validStyles.json';
```

Comment:
Another reason to use a configuration task is to consolidate declarations of this name, since [it's currently duplicated here](https://github.com/mozilla/fxa-content-server-l10n/pull/63/files#diff-48bf94fdac401ff9cacb51d8adc46841R7).